buildSrc/src/main/groovy/airbyte-source-test.gradle

```diff
@@ -27,11 +27,19 @@ class AirbyteSourceTestPlugin implements Plugin<Project> {
                             '--pythonContainerName', pythonContainerName
                 }
             }
+
+            outputs.upToDateWhen { false }
```

Comment:
what does that do?

Reply:
It makes it so the standard source test is always re-run. Since it depends on an external resource, even if all of the upstream source files haven't changed, we should re-run it since it is also checking our compatibility with an external api.